src/components/Timer/Timer.js

```diff
@@ -1,23 +1,43 @@
+import { useState } from 'react';
 import useTimer from './../../hooks/useTimer';
 import fromSecToMin from '../../utils/fromSecToMin';
+import TimerInput from './../Input/TimerInput';
 
-const POMODORO = 10;
-const SHORT_BREAK = 5;
-const LONG_BREAK = 20;
-
-const intervalOrder = [
-    POMODORO,
-    SHORT_BREAK,
-    POMODORO,
-    SHORT_BREAK,
-    POMODORO,
-    SHORT_BREAK,
-    POMODORO,
-    LONG_BREAK,
-];
+const invalidChars = ['-', '+', 'e', 'E'];
 
 const Timer = () => {
-    const { timer, isRunning, setIsRunning } = useTimer(intervalOrder);
+    const [pomodoro, setPomodoro] = useState(60);
+    const [validatePomodoroInput, setValidatePomodoroInput] = useState(60);
+    const [shortBreak, setShortBreak] = useState(10);
+    const [validateShortBreakInput, setValidateShortBreakInput] = useState(10);
+    const [longBreak, setLongBreak] = useState(20);
+    const [validateLongBreakInput, setValidateLongBreakInput] = useState(20);
```

Comment:
here it seems we have too many states in one component.

I know it's because of the form but maybe we could use **uncontrolled inputs** instead of **controlled inputs**?

And then place the validation logic in the form, and check for it `onSubmit`.

You can have a look into HTML form validation, I guess you could use the `pattern` input attribute here?

For example `pattern="[1-9]|[1-5][0-9]|60"` would tell if the number is between 1 and 60

Here is some docs: https://developer.mozilla.org/en-US/docs/Learn/Forms/Form_validation